ynput/ayon-core
# Chore: Move Harmony client code

## Changelog Description
Moved harmony client code next to server codebase.

## Additional info
Change file structure of `stdout_broker` tool. Class `StdOutBroken` moved to `broker.py` and added it to `__init__.py` of the tool.

## Testing notes:
1. Create package of harmony and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Harmony should work as before.
